internal/commands/app/create_inputs.go

```diff
@@ -153,55 +154,110 @@ func (i *createInputs) resolveLocalPath(ui terminal.UI, wd string) (string, erro
 }
 
 func (i *createInputs) resolveClusters(ui terminal.UI, client atlas.Client, groupID string) ([]dataSourceCluster, []string, error) {
-	clusters, err := client.Clusters(groupID)
-	if err != nil {
-		return nil, nil, err
-	}
+	if i.Template != "" {
```

Comment:
decided to do some similar "if creating template app vs default app" logic splitting here as well